ci_framework/roles/libvirt_manager/templates/domain.xml.j2

```diff
@@ -0,0 +1,106 @@
+<domain type='kvm'>
+  <name>cifmw-{{ vm_type }}-{{ vm_id }}</name>
+  <memory unit='GB'>{{ vm_data.value.memory | default(2) }}</memory>
+  <vcpu placement='static'>{{ vm_data.value.cpu | default(2) }}</vcpu>
+  <os>
+    <type arch='x86_64' machine='pc-q35-rhel9.2.0'>hvm</type>
```

Comment:
IMHO.

Try not specify exact machine, q35 is the generic machine type.

Reasonable to expect the default works, the exact might have issues on the long run.

Reply:
indeed - updated.